p3-frontend/src/app/app.component.spec.ts

```diff
@@ -25,11 +25,12 @@ describe('AppComponent', () => {
     const app = fixture.componentInstance;
     expect(app.title).toEqual('p3-frontend');
   });
-
+/*
```

Comment:
don't comment them out, do it like daniel did here https://github.com/1054-Daniel-Felleman-Revature-Swag-Shop/RevatureShopFrontend/pull/1
that way they're at least more visible and are still getting at least compiled and listed as tests for the devs to see.

Reply:
resolve this conversation once you've addressed it with either a reply or with code